Docs/Luadoc/Lua.xml

```diff
@@ -968,6 +970,7 @@
 		<Class base='ActorFrame' name='NoteField'>
 			<Function name='DidHoldNote'/>
 			<Function name='DidTapNote'/>
+			<Function name='get_column_actors'/>

```

Comment:
I prefer the underscore style because I find it easier to read the words in a name, so I named all new functions for this PR in that style.  I'm probably going to rename the other NoteField lua functions to match, which should be fine since they're so new it's likely nobody besides me has used them.